modules/docs/mdx/CONTRIBUTING.mdx

```diff
@@ -9,42 +9,37 @@ part of the community. Before you contribute, please take a moment and look thro
 sections:
 
 - [Code of Conduct](#code-of-conduct)
-- [Contributor License Agreement](#contributor-license-agreement)
-- [Contributing Guidelines](#contributing-guidelines)
-  - [How to Contribute](#how-to-contribute)
+- [How to Contribute](#how-to-contribute)
+  - [Communicate on Issues](#communicate-on-issues)
   - [Git Guidelines](#git-guidelines)
-- [Getting Started](#getting-started)
+    - [Branches](#branches)
+    - [Commit Message Format](#commit-message-format)
+    - [Commit Descriptions](#commit-descriptions)
+    - [Pull Requests](#pull-requests)
+  - [Developer Experience](#developer-experience)
+    - [Yarn and Workspaces](#yarn-and-workspaces)
+    - [Storybook](#storybook)
+    - [Testing](#testing)
+    - [Automation on Commit](#automation-on-commit)
+    - [Releases](#releases)
+- [Getting Started Developing](#getting-started-developing)
   - [Creating a Module](#creating-a-module)
   - [Exporting a Module](#exporting-a-module)
   - [Building Modules](#building-modules)
   - [Testing Modules](#testing-modules)
   - [Code Style Guide](#code-style-guide)
-  - [Canvas Kit Labs & Preview](#canvas-kit-labs-preview)
+  - [Canvas Kit Labs & Preview](#canvas-kit-labs--preview)
   - [Editors](#editors)
+- [Contributor License Agreement](#contributor-license-agreement)
 
 ## Code of Conduct
 
 At Workday, we are committed to a culture of integrity, innovation, and fun. That culture extends to
 the community that we are building here through Canvas. By participating in it, you are expected to
-uphold this [code of conduct](https://github.com/Workday/canvas-kit/tree/master/CODE_OF_CONDUCT.md)
-and agree to our CLA.
-
-## Contributor License Agreement
```

Comment:
I moved this down to keep it but push the relevant content up (and keep the audience reading).